src/components/viewer/GalleryViewer/index.js

```diff
@@ -155,9 +159,26 @@ export default class GalleryViewer extends React.PureComponent {
         super(props);
 
         this.state = {
+            isFullscreen: false,
             screenshotMode: false,
             currentScreenshot: undefined,
         };
+
+        this.viewerRef = React.createRef();
+    }
+
+    componentDidMount() {
+        document.addEventListener('fullscreenchange', this.handleFullscreenChange);
+    }
+
+    componentWillUnmount() {
+        document.removeEventListener('fullscreenchange', this.handleFullscreenChange);
+    }
```

Comment:
Why do this?
isFullscreen is set as false initially.
isFullscreen's value is irrelevant when a component dismounts.

Reply:
This is to remove the event listener from document